samples/drivers/video/capture/README.rst

```diff
@@ -24,6 +24,9 @@ Supported camera modules on some i.MX RT boards can be found below.
 - :ref:`mimxrt1170_evk`
 - `OV5640 camera module`_
 
+Also Arduino Nicla Vision can be used in this sample as capture device, in that case
+The user can transfer the captured frames through on board USB.
+
```

Comment:
In case you also wanted a link to the board, you can use a reference to [here](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arduino/nicla_vision/doc/index.rst?plain=1#L1)
```suggestion
- :ref:`arduino_nicla_vision_board`
```

Reply:
Hmm I was sure I tested using that way, and the documentation build has  been failed.